github-extension/src/browser/github-frontend-contribution.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { injectable, inject } from 'inversify';
+import { StatusBar, StatusBarAlignment } from '@theia/core/lib/browser/status-bar/status-bar';
+import { GithubService } from '../common/github-service';
+import { GitHubProperties } from './github-properties';
+import { User } from '../common/github-model';
+import { GITHUB_COMMANDS } from './github-commands';
+import { FrontendApplicationContribution, FrontendApplication } from '@theia/core/lib/browser';
+
+@injectable()
+export class GitHubFrontendContribution implements FrontendApplicationContribution {
+
+    constructor(
+        @inject(StatusBar) protected readonly statusBar: StatusBar,
+        @inject(GitHubProperties) protected readonly gitHubProperties: GitHubProperties,
+        @inject(GithubService) protected readonly gitHubService: GithubService
+    ) {
+        this.gitHubProperties.onPropertiesChange(() => {
```

Comment:
As @azatsarynnyy commented, this should be moved to `onStart()`. It might cause unexpected behavior by set the handler before called `onStart()`.